src/core/mount.c

```diff
@@ -1891,6 +1892,9 @@ static void mount_enumerate(Manager *m) {
 
         assert(m);
 
+        if (MANAGER_IS_USER(m) && getenv_bool("SYSTEMD_SKIP_DEV_ENUMERATE") == 1)
+                return;
```

Comment:
i'd drop the conditionalization on the --user mode. if its hidden between an env var, i see no reason not to let people shoot themselves in the foot.

and after all it might make sense to give people control over this in container environments in system mode or so.